pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -37,7 +37,13 @@ func NewFlowRecords() *FlowRecords {
 }
 
 // AddOrUpdateFlowRecord adds or updates the flow record in the record map given the connection.
```

Comment:
Can you mention in the comment that the function makes a copy of the connection object, to avoid race conditions between the connection store and the flow exporter?

Reply:
Updated. Thanks!